### PR TITLE
chore: audit package.json's

### LINKED ISSUE #1307: audit package.json(s)
and only keep packages that are used

- [ ] backend
- [ ] auth
- [ ] token-introspection
- [ ] mock-account-servicing-entity
- [ ] local-http-signature
- [x] frontend

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- I did a [depcheck](https://github.com/depcheck/depcheck) on all packages listed in #1307 
- I made sure that build, test, and generate commands still work

## Context

- fixes #1307 

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

**Comment by sabineschaller** [comment]
Types generated using 
`pnpm --filter token-introspection generate:types`
seem to break the `token-introspection` package.

**Comment by mkurapov** [comment]
@sabineschaller 
I think the newer versions of `openapi-typescript` are the issue. I'll need to try and see whether
- https://github.com/interledger/open-payments/issues/249 
is resolved, and what the breaking changes are.

File: packages/backend/package.json [hunk 1]
```diff
@@ -82,9 +74,7 @@
     "objection": "^3.1.1",
     "objection-db-errors": "^1.1.2",
     "oer-utils": "5.1.3-alpha.2",
-    "pg": "^8.11.2",
     "pino": "^8.15.0",
-    "pino-pretty": "^10.2.0",
```
Comment (BlairCurrey) [inline packages/backend/package.json:87]: I believe this is still being used here: https://github.com/interledger/rafiki/blob/main/packages/backend/src/tests/app.ts#L43

[the remark this answers is not in the document]
Comment (raducristianpopa) [reply packages/backend/package.json:87]: Maybe it can be moved to `devDependecies` ?

**Review by njlie — DISMISSED** [review]
AS package changes LGTM

**Review by mkurapov — COMMENTED** [review]
Other than `pg`, and moving `pino-pretty` to devDependencies, looks good

**Review by raducristianpopa — APPROVED** [review]

**Review by mkurapov — APPROVED** [review]

**Pull request merged**